### DISCUSSION TITLE
html script tag in page is ignored

### DISCUSSION THREAD

**Discussion by denispeyrusaubes** [McShelby/hugo-theme-relearn#369] [Q&A] [answered]
Hi,

I try to embed a snippet of code manager by gitlab using `<script src="https://gitlab.com/-/snippets/1963638.js"></script>`.
But nothing appears in my page. I use to embed such code using another Hugo template, with success. Do you hava an idea why it does not appear here ?

Thanks for your help,
D.

**Comment by McShelby** [comment, marked as answer]
https://mcshelby.github.io/hugo-theme-relearn/basics/customization/ `custom-header.html` is the way to go here.
